src/Deprecated/Engine/Shared/FileUtilities.cs

```diff
@@ -19,6 +19,13 @@ namespace Microsoft.Build.BuildEngine.Shared
     /// <owner>SumedhK</owner>
     static internal class FileUtilities
     {
+        internal static bool EnsurePathIsSingleQuoted(string path)
+        {
```

Comment:
I wouldn't bother with modifying this version. It isn't really used.

Reply:
Shoot.  Yeah I meant to wipe that out.  We were having a issue with accessing the method we created and we were spitballing on why.  And we added this to deprecated.  Thought it might have had something to do with it being a partial class.